Makefile

```diff
@@ -138,5 +138,11 @@ build-images: build-rpms
 .PHONY: build-images
 
 update-deps:
-	glide update --skip-test -v
+	go get -d -u k8s.io/api@kubernetes-1.13.5
+	go get -d -u k8s.io/apimachinery@kubernetes-1.13.5
+	go get -d -u k8s.io/apiserver@kubernetes-1.13.5
+	go get -d -u k8s.io/client-go@kubernetes-1.13.5
+	go mod edit -replace=github.com/docker/distribution=github.com/openshift/docker-distribution@image-registry-4.0-distribution-16128bb
```

Comment:
@dmage shouldn't we also update openshift api and client-go?

```
go get -d -u github.com/openshift/api@master
go get -d -u github.com/openshift/client-go@master
```

Reply:
@adambkaplan I've updated `update-deps` target, now it really updates dependencies. We can't use master until we rebase, so I pinned `release-4.1`.